letsencrypt/cli.py

```diff
@@ -483,6 +587,9 @@ def create_parser(plugins, args):
     #for subparser in parser_run, parser_auth, parser_install:
     #    subparser.add_argument("domains", nargs="*", metavar="domain")
     helpful.add(None, "-d", "--domains", metavar="DOMAIN", action="append")
+    helpful.add(
+        None, "-D", "--duplicate", dest="duplicate", action="store_true",
```

Comment:
Isn't -D the short flag? Or are you saying that maybe there shouldn't be a short flag?